docker/gcloud_build.sh

```diff
@@ -4,11 +4,16 @@ set -ex
 
 gcloud docker --authorize-only
 
+if [ -z $DOCKER_TAG ]
+then
+    export DOCKER_TAG=experiment
+fi
+
 if [ -z $BAZEL_OUTBASE ]
 then
-    bazel run //docker:mixer gcr.io/$PROJECT/mixer:experiment
+    bazel run //docker:mixer gcr.io/$PROJECT/mixer:$DOCKER_TAG
 else
-    bazel --output_base=$BAZEL_OUTBASE run //docker:mixer gcr.io/$PROJECT/mixer:experiment
+    bazel --output_base=$BAZEL_OUTBASE run //docker:mixer gcr.io/$PROJECT/mixer:$DOCKER_TAG
 fi
 
```

Comment:
I think while pulling the build, we still want to be able to say

gcr.io/$PROJECT/mixer:latest   otherwise I have to lookup sha everytime.